cumulus/pallets/collator-selection/src/lib.rs

```diff
@@ -982,3 +985,16 @@ pub mod pallet {
 		}
 	}
 }
+
+/// TypedGet implementaion to get the AccountId of the StakingPot.
+/// QUESTION: is there a better way to do this?
```

Comment:
You could implement it directly on the pallet, but that would be quite unclean. I think there no better way than this.